packages/integrations/cloudflare/src/index.ts

```diff
@@ -205,7 +205,8 @@ export default function createIntegration(args?: Options): AstroIntegration {
 				if (isModeDirectory) {
 					const functionsUrl = new URL('functions', _config.root);
```

Comment:
actually it is enough to add a trailing slash behind `functions` this you can fix the issue easily

Reply:
Aww, thanks for the guidance 😊